### DISCUSSION TITLE
EFN shows SUA - what impacts for me?

### DISCUSSION THREAD

**Discussion by RobeKrau** [Akaflieg-Freiburg/enroute#396] [Q&A] [answered]
EFN shows information referring to a map area like "SUA FIS LANGEN 128.875 FL 100 / GND".

While I am familiar with ED-R and ED-D areas, I did not meet SUA in German airspace yet.

Does anyone know details about this?

**Comment by kebekus** [comment, marked as answer]
@RobeKrau You can ignore this airspace. These are sectors for the flight information service, which <re unfortunately displayed twice at the moment. Reason: Changes in the openAIP database that I was not informed about. I will take care of the problem over the holidays. Until then, I apologize for the confusion.

**Reply by RobeKrau** [reply]
@kebekus Thank you, Stefan, for your quick reply!
